ui/pages/login/callback.js

```diff
@@ -28,8 +34,20 @@ export default function Callback() {
       } else {
         router.replace('/')
       }
-
       window.localStorage.removeItem('next')
+
+      let visitedOrgs = cookies.get('orgs') || []
+      if (visitedOrgs.findIndex(x => x.url === window.location.origin) === -1) {
```

Comment:
Would change this to `visitedOrgs.find` instead of `findIndex` since it's easier to read